en/docs/setup/token-persistence.md

```diff
@@ -9,24 +9,20 @@ database (synchronous and asynchronous token persistence).
 The following sections guide you through the difference between these
 two approaches and how to configure them.
 
--   [Synchronous token
-    persistence](#TokenPersistence-Synchronoustokenpersistence)
--   [Asynchronous token
-    persistence](#TokenPersistence-Asynchronoustokenpersistence)
--   [Recovery flow for token
-    persistence](#TokenPersistence-Recoveryflowfortokenpersistence)
+-   [Synchronous token persistence](#synchronous-token-persistence)
+-   [Asynchronous token persistence](#asynchronous-token-persistence)
+-   [Recovery flow for token persistence](#recovery-flow-for-token-persistence)
```

Comment:
Let's remote the TOC as there is right-hand navigation for the sub-titles

Reply:
Fixed: 69387cab7c97dd931cffa7a97e11d3abc27ef908